.github/workflows/code_coverage.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - name: Update toolchain
         run: rustup update
       - name: Test
-        run: cargo test --features all-keys,compiler,esplora,compact_filters --no-default-features
+        run: cargo test --features all-keys,compiler,esplora,ureq,compact_filters --no-default-features
```

Comment:
why does this flag need to be here shouldn't it just be ureq if esplora is flagged but `async-interface` is not?

Reply:
(oh wait now I get it).